Installation not working on Fedora

**WHAT:**

On Fedora 25, when installing the dependencies with `pip install -r requirements.txt`, the installation of `librabbitmq` fails with the following error:

```
clib/librabbitmq/amqp_socket.c: In function ‘amqp_login_inner’:
  clib/librabbitmq/amqp_socket.c:1195:28: error: ‘AMQ_PLATFORM’ undeclared (first use in this function)
           amqp_cstring_bytes(AMQ_PLATFORM);
                              ^~~~~~~~~~~~
  clib/librabbitmq/amqp_socket.c:1195:28: note: each undeclared identifier is reported only once for each function it appears in
  error: command 'gcc' failed with exit status 1
  
  ----------------------------------------
  Failed building wheel for librabbitmq
  Running setup.py clean for librabbitmq
Failed to build librabbitmq

```
You can find a more detailed version of the output in [this text-dump](https://paste.opensuse.org/view/simple/22524203).


**HOW TO REPRODUCE:**

On any Fedora system, navigate to the root of the project, and run `pip install -r requirements.txt`.


**PROPOSED SOLUTION:**

This seems like an issue with version `1.6.1` of `librabbitmq` on Fedora. Switching to `1.5.1` leads to a successful install. Possible solutions:

 - Edit `requirements/common.txt` to use the working version:
    ```
    librabbitmq==1.5.1
    ```

 - Add a new Fedora-specific `requirements-fedora.txt` file with this change.

 - Or simply,
    ```
    pip install librabbitmq==1.5.1
    ```

**ALSO SEE:**

https://github.com/fossasia/open-event-orga-server/pull/2443

--- a/requirements/common.txt
+++ b/requirements/common.txt
@@ -21,7 +21,7 @@ celery>=3.1.23,<4
 redis>=2.10.5,<3
 apscheduler>=3.3.0,<4
 pillow>=3.4.2,<3.5
-librabbitmq>=1.6.1,<2
+librabbitmq==1.5.1
 gunicorn>=19.6.0,<20
 boto>=2.45.0,<3
 geoip2>=2.4.2,<3

